<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>end_sequenceDialog</class>
 <widget class="QDialog" name="end_sequenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finalise sequence:</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="searchPage">
      <attribute name="title">
       <string>Add sequence details</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>291</width>
         <height>501</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QPushButton" name="endSequenceButton">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Notes:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="sequenceNotesPlainTextEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="studyLabel">
          <property name="text">
           <string>Study:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="findDigitLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Digit imaged:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_33">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Left</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_34">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Right</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QRadioButton" name="otherDigitRadioButton">
             <property name="text">
              <string>Other object imaged</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QRadioButton" name="RD1radioButton">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="LD1radioButton">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QRadioButton" name="LD2radioButton">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QRadioButton" name="LD3radioButton">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QRadioButton" name="RD2radioButton">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QRadioButton" name="RD3radioButton">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QRadioButton" name="RD4radioButton">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QRadioButton" name="LD4radioButton">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QRadioButton" name="LD5radioButton">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QRadioButton" name="RD5radioButton">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="sequenceNameLabel">
          <property name="text">
           <string>Sequence name:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="sequenceNameLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="qualityCheckBox">
          <property name="text">
           <string>Quality acceptable</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
